src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -2533,36 +2516,30 @@ def total_BEC_eq(c):
     def display_flowsheet_cost(b):
         # This method accepts a flowsheet-level costing block
         print("\n")
-        print("Total bare erected cost: $%.3f Million" % value(b.total_BEC))
+        print("Total bare erected cost: %.3f" % value(b.total_BEC))
         if hasattr(b, "total_overnight_capital"):
             print(
-                "Total overnight (installed) equipment cost: $%.3f Million"
+                "Total overnight (installed) equipment cost: %.3f"
                 % value(b.total_overnight_capital)
             )
         if hasattr(b, "annualized_cost"):
-            print(
-                "Total annualized capital cost: $%.3f Million"
-                % value(b.annualized_cost)
-            )
+            print("Total annualized capital cost: %.3f" % value(b.annualized_cost))
         print()
         if hasattr(b, "total_fixed_OM_cost"):
-            print(
-                "Total annual fixed O&M cost: $%.3f Million"
-                % value(b.total_fixed_OM_cost)
-            )
+            print("Total annual fixed O&M cost: %.3f" % value(b.total_fixed_OM_cost))
         if hasattr(b, "total_variable_OM_cost"):
             print(
-                "Total annual variable O&M cost: $%.3f Million"
+                "Total annual variable O&M cost: %.3f"
```

Comment:
Another minor note, but you might want to consider `fstrings` for this (I find them easier to read).